### ISSUE TITLE
Infinite loops in multiple invoices creation

### ISSUE THREAD

**Issue by alxster** [odoo/odoo#6622]
Impacted versions:
- 8.0

Steps to reproduce:

Create a customer and 2 invoices to that customer from source code (no interface)

Current behavior:

It loops during the creation of the second invoice

Expected behavior:

It should create both invoices

I am developing a functionality to import a large file of invoices grouped by customer.
For each customer it is created and then all the invoices created and set to that partner.
Problem is the commercial_partner_id field, which is computed into res.partner and related to this in account.invoice
Basically, the first invoice is correctly saved, but when I try to save the second one, when the account_invoice.partner_id is set, the partner_id object is triggered to recompute.
This provokes that self.all.todo dictionary is filled also with the instance of the first invoice, thus the while loop into recompute method for second invoice never ends.

**Comment by Yenthe666** [comment]
Hi @alxster,

Thank you for your bug report and our apoligies that we haven't had the time to look at this bug report. We do our best to handle all of them, but we sometimes miss some.
At this point Odoo 8 is no longer supported as Odoo only supports three versions (9.0, 10.0, 11.0 at the moment).
If you can still reproduce this issue on Odoo 9, Odoo 10 or Odoo 11 we encourage you to create a new bug report.

Regards,
Yenthe
